tools/cxbe/Cdxt.cpp

```diff
@@ -0,0 +1,97 @@
+// Licensed under GPLv2 or (at your option) any later version.
+// Copyright (C) 2020 Jannik Vogel
+
+#include "Exe.h"
+#include "Common.h"
+
+#include <string.h>
+#include <stdio.h>
+
+// program entry point
+int main(int argc, char *argv[])
+{
+    char szErrorMessage[ERROR_LEN+1] = {0};
+    char szExeFilename[OPTION_LEN+1] = {0};
+    char szDxtFilename[OPTION_LEN+1] = {0};
+
+    const char* program = argv[0];
+    const char* program_desc = "CDXT: EXE to DXT Relinker";
+    static Option options[] = {
+        { szExeFilename, NULL,  "exefile"  },
+        { szDxtFilename, "OUT", "filename" },
+        { NULL }
+    };
+
+    if(ParseOptions(argv, argc, options, szErrorMessage)) {
+        goto cleanup;
+    }
+
+    // verify we recieved the required parameters
+    if(szExeFilename[0] == '\0')
+    {
+        ShowUsage(program, program_desc, options);
+        return 1;
+    }
+
+    // if we don't have an Dxt filename, generate one from szExeFilename
+    if(szDxtFilename[0] == '\0')
+    {
+        GenerateFilename(szDxtFilename, ".dxt", szExeFilename, ".exe");
+    }
+
+    // open and convert Exe file
+    {
+        Exe *ExeFile = new Exe(szExeFilename);
+
+        if(ExeFile->GetError() != 0)
+        {
+            strncpy(szErrorMessage, ExeFile->GetError(), ERROR_LEN);
+            goto cleanup;
+        }
+
+        // Set up subsystem (will be ignored)
+        ExeFile->m_OptionalHeader.m_subsystem_version_major = 1;
+        ExeFile->m_OptionalHeader.m_subsystem_version_minor = 0;
+        ExeFile->m_OptionalHeader.m_subsystem = IMAGE_SUBSYSTEM_XBOX;
+
+        // Set up alignments (DXTs typically use 32 byte alignments)
+        //FIXME: Report error if FileAlignment > SectionAlignment
+        ExeFile->m_OptionalHeader.m_file_alignment =
+            ExeFile->m_OptionalHeader.m_section_alignment;
+
```

Comment:
Force existence of import table.
Currently forced by explicitly importing something in user-code.
Will probably forced by auto-linked dummy in the future.
Could also be forced here (for users who don't link with nxdk).